### PR TITLE
Tutorials

### PR DESCRIPTION
**Pull request by anushreeSonak**
Hi @amitsahu8 
Please Find stack and HashMap programs
HashMap- To calculate the size of the Map
Stack- Push and pop the elements from the stack

### CODE REVIEW

File: Country.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package firstPackage;
```
Comment (vrushalee123) [inline Country.java:1]: Please do not include .class file in pull request. Add it in .gitignore.

File: Country.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		
+		//Print destination City
+		System.out.println(destination);
+	}
+
+	
+	public static String findDestination(List<List<String>> paths) 
+	{
+		//Declare set variable
+		Set<String> startCities = new HashSet<>();
```
Comment (vrushalee123) [inline Country.java:28]: Make it private.

File: First/.classpath [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (vrushalee123) [inline First/.classpath:1]: do not include .classpath

File: Example2.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
+package firstPackage;
+
```
Comment (vrushalee123) [inline Example2.java:2]: Always write problem statement in comment.

File: First/.project [hunk 5]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (vrushalee123) [inline First/.project:1]: Do not include .project

Comment (vrushalee123) [reply First/.project:1]: Add it in .gitignore

**Review by vrushalee123 — CHANGES_REQUESTED** [review]

File: Country.java (hunk 1 above)
Comment (anushreeSonak) [reply Country.java:1]: Done !

File: First/.classpath (hunk 3 above)
Comment (anushreeSonak) [reply First/.classpath:1]: Done!

File: First/.project (hunk 5 above)
Comment (anushreeSonak) [reply First/.project:1]: Done !

File: Country.java (hunk 2 above)
Comment (anushreeSonak) [reply Country.java:28]: Uploaded new file for this i.e.  Findestination.java

**Pull request merged**